lib/server/socket.js

```diff
@@ -12,10 +12,10 @@ module.exports = ({ server, filters, config }) => {
   const ioConfig = {
     transformer: 'engine.io',
     parser: 'EJSON',
-    maxLength: parseInt(config.socketMaxResponseLength) ?? 20971520, // support up to 20MB in response bodies
+    maxLength: config.socketMaxResponseLength ?? 20971520, // support up to 20MB in response bodies
```

Comment:
We could possibly have used `parseInt(config.socketMaxResponseLength ?? 20971520)`